### PR TITLE
VIH-7766 Create Endpoint which takes in a full list of participants and updates the conference with them

### PR DESCRIPTION
**Pull request by AWildBugAppeared**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-7766

### Change description ###
Created a new endpoint and command which is responsible for updating a conference's participants.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

**Review by alberto-hortiguela — CHANGES_REQUESTED** [review]
Comments for review

File: VideoApi/VideoApi.IntegrationTests/Database/Commands/UpdateConferenceParticipantsCommandTests.cs [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+            });
+
+            var command = BuildCommand();
+
+            //Act/Assert
+            Assert.ThrowsAsync<ParticipantLinkException>(() => _handler.Handle(command));
+        }
+
+        [Test]
+        public async Task Should_remove_participants_from_conference()
```
[the remark this answers is not in the document]
Comment (AWildBugAppeared) [reply VideoApi/VideoApi.IntegrationTests/Database/Commands/UpdateConferenceParticipantsCommandTests.cs:199]: Spoken with Alberto. This test is required.

**Review by alberto-hortiguela — APPROVED** [review]
LGTM

**Pull request merged**